client/my-sites/checkout/composite-checkout/payment-methods/paypal.tsx

```diff
@@ -229,6 +230,7 @@ function PayPalSubmitButton( {
 			buttonType="paypal"
 			isBusy={ FormStatus.SUBMITTING === formStatus }
 			fullWidth
+			aria-label={ i18n.translate( 'Pay with PayPal' ) }
```

Comment:
It's not a big deal, but since we're already using `useI18n()` in this file, it might be worth calling it in this function and using `__()` here instead of having two translation packages for these components.

https://github.com/Automattic/wp-calypso/blob/daf4ea960e4ed21ff48927085f86f39bddaa1074/client/my-sites/checkout/composite-checkout/payment-methods/paypal.tsx#L245-L248

Reply:
Good point! I updated the code accordingly.